lib/configuration.yaml

```diff
@@ -14979,6 +14979,13 @@ mainnet_dryrun_full: &mainnet_dryrun_full
     attribResrictEpoch: 2
     addrAttribSize: 128
     txAttribSize: 128
+  update: &mainnet_base_update
```

Comment:
It seems that this is a duplicate anchor as there's already a  `&mainnet_base_update` anchor defined under the `mainnet_base` key. We should probably remove the `&mainnet_base_update` anchor here.

Reply:
Also, I believe we could just do something like this:
```
  update:
    <<: *mainnet_base_update
    lastKnownBlockVersion:
      bvMajor: 1
      bvMinor: 0
      bvAlt: 0
```
Since the only difference from `&mainnet_base_update` is the `lastKnownBlockVersion`.